web/js/MapUI/Components/CreateMap/CreateMap.js

```diff
@@ -0,0 +1,278 @@
+    setUI(uiCopy);
+  });
+
+  /**
+   * Create map object
+   *
+   * @method createMap
+   * @static
+   *
+   * @param {object} proj - Projection properties
+   * @param {object} dateSelected
+   *
+   * @returns {object} OpenLayers Map Object
+   */
+  const mapCreation = (proj, uiCopy) => {
+    const mapContainerEl = document.getElementById('wv-map');
+    const mapEl = document.createElement('div');
+    const id = `wv-map-${proj.id}`;
+
+    mapEl.setAttribute('id', id);
+    mapEl.setAttribute('data-proj', proj.id);
+    mapEl.classList.add('wv-map');
+    mapEl.style.display = 'none';
+    mapContainerEl.insertAdjacentElement('afterbegin', mapEl);
+
+    const scaleMetric = new OlControlScaleLine({
+      className: 'wv-map-scale-metric',
+      units: 'metric',
+    });
+    const scaleImperial = new OlControlScaleLine({
+      className: 'wv-map-scale-imperial',
+      units: 'imperial',
+    });
+    const rotateInteraction = new OlInteractionDragRotate({
+      condition: altKeyOnly,
+      duration: animationDuration,
+    });
+    const mobileRotation = new OlInteractionPinchRotate({
+      duration: animationDuration,
+    });
+    const map = new OlMap({
+      view: new OlView({
+        maxResolution: proj.resolutions[0],
+        projection: olProj.get(proj.crs),
+        center: proj.startCenter,
+        zoom: proj.startZoom,
+        maxZoom: proj.numZoomLevels,
+        enableRotation: true,
+        extent: proj.id === 'geographic' ? [-250, -90, 250, 90] : proj.maxExtent,
+        constrainOnlyCenter: true,
+      }),
+      target: id,
+      renderer: ['canvas'],
+      logo: false,
+      controls: [scaleMetric, scaleImperial],
+      interactions: [
+        doubleClickZoom,
+        new OlInteractionDragPan({
+          kinetic: new OlKinetic(-0.005, 0.05, 100),
+        }),
+        new OlInteractionPinchZoom({
+          duration: animationDuration,
+        }),
+        new OlInteractionMouseWheelZoom({
+          duration: animationDuration,
+        }),
+        new OlInteractionDragZoom({
+          duration: animationDuration,
+        }),
+      ],
+      loadTilesWhileAnimating: true,
+      loadTilesWhileInteracting: true,
+      maxTilesLoading: 32,
+    });
+    map.wv = {
+      scaleMetric,
+      scaleImperial,
+    };
+    map.proj = proj.id;
+
+    map.getView().on('change:resolution', () => {
+      events.trigger(MAP_MOVE_START);
+    });
+
+    // This component is inside the map viewport container. Allowing
+    // mouse move events to bubble up displays map coordinates--let those
+    // be blank when over a component.
+    document.querySelector('.wv-map-scale-metric').addEventListener('mousemove', (e) => e.stopPropagation());
+    document.querySelector('.wv-map-scale-imperial').addEventListener('mousemove', (e) => e.stopPropagation());
+
+    // Allow rotation by dragging for polar projections
+    if (proj.id !== 'geographic' && proj.id !== 'webmerc') {
```

Comment:
What is the advantage of `proj.id !== 'geographic' && proj.id !== 'webmerc'` as opposed to `proj.id == 'polar'`?

Reply:
I think you would need to write `proj.id === 'artic' || proj.id === 'antartic'` anyway. It was already written this way prior to when I changed these files so I am just going to leave this one.